context/app/api/vitessce_confs/base_confs.py

```diff
@@ -166,6 +207,7 @@ def get_conf_cells(self):
             raise FileNotFoundError(message)
         vc = VitessceConfig(name="HuBMAP Data Portal")
         dataset = vc.add_dataset(name="Visualization Files")
+        # The sublcass initializes _files in its __init__ method
```

Comment:
This is helpful: From the top, one thing that's hard to get a handle on is which classes, like this one, are abstract. perhaps add `Abstract...` to the class name? There's also python's [`ABC`](https://docs.python.org/3/library/abc.html)... worth reading about, but I don't know that they'd give us much.